docker-compose.yml

```diff
@@ -113,6 +113,7 @@ services:
           -- celery -A iib.workers.tasks worker --loglevel=info
     environment:
       IIB_DEV: 'true'
+      REGISTRY_AUTH_FILE: '/root/.docker/config.json'
```

Comment:
Just question: It did not work without it? 
I thought that container tools are looking for this path by default.

Reply:
But it didn't work properly but it does now :heart: